test/api/test_rest.py

```diff
@@ -471,6 +471,14 @@ def test_testruns(self):
         data = self.hit('/api/testruns/%d/' % self.testrun.id)
         self.assertEqual(self.testrun.id, data['id'])
 
+    def test_testruns_null_metrics_attr(self):
```

Comment:
Cool test. One more thing though: could you make one test where there are actual metrics for a testrun? This would test the other way around, when 'metrics' and 'metrics_file' attributes aren't `None`

Reply:
Done!